doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -4962,214 +4964,342 @@ \subsubsection{Inputs}\label{inputs-15-010}
+
+\subsubsection{Inputs}\label{wwhpeir_inputs}
+
+\paragraph{Field: Name}\label{wwhpeir_inputs_name}
+
+This alpha field contains the identifying name for the ground source heat pump.
+
+\paragraph{Field: Load Side Inlet Node Name}\label{wwhpeir_inputs_load_inlet_node}
+
+This alpha field contains the heat pump's load side inlet node name.
+
+\paragraph{Field: Load Side Outlet Node Name}\label{wwhpeir_inputs_load_outlet_node}
+
+This alpha field contains the heat pump's load side outlet node name.
+
+\paragraph{Field: Source Side Inlet Node Name}\label{wwhpeir_inputs_source_inlet_node}
+
+This alpha field contains the heat pump's source side inlet node name.
+
+\paragraph{Field: Source Side Outlet Node Name}\label{wwhpeir_inputs_source_outlet_node}
+
+This alpha field contains the heat pump's source side outlet node name.
+
+\paragraph{Field: Companion Heat Pump Name}\label{wwhp_eir_inputs_companion_name}
+
+This optional alpha field contains the name of a companion heat pump object.  If the current object is heating type then this is expected to include the paired cooling object, and vice-versa.  The companion is used for sizing if plant sizing data is unavailable, and during the simulation to ensure that any unexpected simultaneous operation of the two objects provides a warning to the user.
+
+\paragraph{Field: Load Side Reference Flow Rate}\label{wwhp_eir_inputs_load_side_flow}
+
+This autosizable field defines the reference load side flow rate for the coil.  This field is used as the nominal flow request for this heat pump unit during the simulation.  The units for this field are [m3/s].
+
+\paragraph{Field: Source Side Reference Flow Rate}\label{wwhp_eir_inputs_source_side_flow}
+
+This autosizable field defines the reference demand side flow rate for the coil.  This field is used as the nominal flow request for this heat pump unit during the simulation.  The units for this field are [m3/s].
+
+\paragraph{Field: Reference Capacity}\label{wwhp_eir_inputs_reference_capacity}
+
+This autosizable field defines the reference load side heat capacity for the coil.  Cooling capacity for a cooling coil and heating capacity for a heating coil.  The units for this field are [W].
+
+\paragraph{Field: Reference Coefficient of Performance}\label{wwhp_eir_inputs_reference_cop}
+
+This field contains the reference COP for this unit, which is used when calculating the power required for the coil under simulation conditions.  The COP is defined as the load side energy transfer divided by the power input, with dimensionless units of [W/W].
+
+\paragraph{Field: Sizing Factor}\label{wwhp_eir_inputs_sizing_factor}
+
+This field allows the user to scale the sizing of this unit compared to the plant-determined autosized flow and heat capacity.
+
+\paragraph{Field: Capacity Modifier Function of Temperature Curve Name}\label{wwhp_eir_inputs_capft}
```

Comment:
The modifier curve descriptions should explain that the curve value is multiplied by the reference capacity (or EIR), and should also includes [C] units on the input variables, and say "curve or table"

Reply:
Clarified all this in a branch.